README.md

```diff
@@ -34,17 +34,27 @@ Usage
     Example:
       nvmw install v0.6.0          Install a specific version number
       nvmw use v0.6.0              Use the specific version
+      nvmw install iojs/v0.6.0     Install a specific version number of io.js
+      nvmw use iojs/v0.6.0         Use the specific version of io.js
```

Comment:
I think it's helpful to user to existing iojs version like `iojs/v1.0.2` on README.